src/main/k/kore.k

```diff
@@ -2,7 +2,7 @@
 module KORE
   // imports STRING
 
-  syntax Module ::= "module" ModuleName Declarations "endmodule" Attribute
+  syntax Module ::= Attribute "module" ModuleName Declarations "endmodule" Attribute
```

Comment:
See Semantics of K document, section 9.1.6 - 9.1.7, a module has its attributes in the end. A Kore definition consists of multiple modules, and the attribute for the entire Kore definition appear in the beginning.

Reply:
Okay, that makse things clear.